src/rules/requireReturnsDescription.js

```diff
@@ -5,6 +5,9 @@ export default iterateJsdoc(({
   utils
 }) => {
   const targetTagName = utils.getPreferredTagName('returns');
+  if (!targetTagName) {
+    return;
```

Comment:
This guard is duplicated 8 times, it can be avoided it if is placed in `utils.forEachTag()`.

Reply:
Good point. I've changed `forEachTag` to `forEachPreferredTag`.